README.rst

```diff
@@ -1114,10 +1149,32 @@ Admin Multitenancy mixins
         list_filter = [SubnetFilter]
         # other options
 
+Management Commands
+-------------------
+
+``export_users``
+~~~~~~~~~~~~~~~~
+
+This command exports user data to a CSV file, including related data such as organizations.
+
+Example usage:
+
+.. code-block:: shell
+
+    ./manage.py export_users
+
+**Arguments**:
+
+- `--exclude-fields`: Comma-separated list of fields to exclude from the export.
+- `--filename`: Filename for the exported CSV, defaults to "openwisp_exported_users.csv".
+
+You can change the exported fields using the `OPENWISP_USERS_EXPORT_USERS_COMMAND_CONFIG
+<#openwisp_users_export_users_command_config>`_ setting.
```

Comment:
I am not sure I understand the point of this since users can use the `--exclude-fields` flag.
I think it's worth to point out that this setting is meant to be used for advanced customizations.